csunplugged/utils/BaseLoader.py

```diff
@@ -36,29 +44,48 @@ def setup_md_to_html_converter(self):
         """
         templates = self.load_template_files()
         extensions = [
-            'markdown.extensions.fenced_code',
-            'markdown.extensions.codehilite',
-            'markdown.extensions.sane_lists',
-            'markdown.extensions.tables',
+            "markdown.extensions.fenced_code",
+            "markdown.extensions.codehilite",
+            "markdown.extensions.sane_lists",
+            "markdown.extensions.tables",
             mdx_math.MathExtension(enable_dollar_delimiter=True)
         ]
         self.converter = Verto(html_templates=templates, extensions=extensions)
         custom_processors = self.converter.processor_defaults()
-        custom_processors.add('remove-title')
+        custom_processors.add("video")
```

Comment:
I believe this is a pointless statement, as `video` should be enabled by default.